stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/channel/internal/ChannelEntity.kt

```diff
@@ -41,6 +41,7 @@ import java.util.Date
  * @param memberCount Number of members in the channel.
  * @param watcherIds The list of watchers` ids.
  * @param watcherCount Number of watchers in the channel.
+ * @param unreadCount Number of unread messages for the current user.
```

Comment:
I can't find `unreadCount` property inside `ChannelEntity` class 🙂

Reply:
You are right, at the end this calculated value is not needed on DB and we don't store it.
Let me remove this line